### ISSUE TITLE
Related stories - move to bottom & switch to categories

### ISSUE THREAD

**Issue by MirandaEcho** [INN/theme-calhealthreport#10]
ESTIMATE/NOTES: The client has already changed the widget title to "Related Stories".
Modification of how this area works may take 2-6h, depending on how it is built in the theme. Note on this, I believe we should have a fairly simple way to do this. If you look on this post, you will see on the right that there is a sidebar with related stories. 

"What I would like is for those stories to appear at the bottom of the posts instead. I'd like to have this option for all of our stories going forward, so ones with a full width feature image and ones with a regular sized feature image"

**Comment by benlk** [comment]
This doesn't apply to posts with the "Hide Sidebar" layout option set.

There is already a "Related Stories" section in the post footer:

![Screen Shot 2020-05-20 at 23 46 19](https://user-images.githubusercontent.com/1754187/82521274-33cd1f00-9af4-11ea-879d-96c2fd81669e.png)

This isn't a widget area; it's powered by the theme.

https://github.com/INN/theme-calhealthreport/blob/68e12f0a99ca2d28d08d2b8aacee07ed04b31353/single.php#L80

https://github.com/INN/theme-calhealthreport/blob/master/template-parts/related-posts.php

Unfortunately, it's hidden on mobile.


If they specifically want to have the widget powering that sidebar to be present at the bottom of the posts, this is what that same "Related Posts by Taxonomy" widget looks like in the "Pre-Footer 1 widgets" widget area, with the same settings as in the sidebar:

![Screen Shot 2020-05-20 at 23 51 21](https://user-images.githubusercontent.com/1754187/82521566-059c0f00-9af5-11ea-8683-aa9ec8fc267c.png)

**Comment by benlk** [comment]
Questions:
- is the existing "Related Stories" block sufficient for their purposes if it's unhidden on mobile?
- if not, do they want to remove that block and replace it with a widget that's styled similarly?

**Comment by MirandaEcho** [comment]
This is the related stories she's referring to:
![Screen Shot 2020-05-22 at 6 20 41 PM](https://user-images.githubusercontent.com/48263361/82713439-fcc44e00-9c58-11ea-8073-d8be9a926b8a.png)

I will clarify if that should replace the one from the theme that's currently at the bottom.

**Comment by MirandaEcho** [comment]
Confirmed, we want to remove the existing "Recommended for you" widget thing at the bottom and instead replace it with the Related Stories widget from the sidebar (pictured above), using categories and not tags.

**Comment by benlk** [comment]
So the remaining tasks here are:
- [x] remove the existing item
- [x] test the widget in the article-bottom widget area added in #17
- [x] styles as appropriate

Because we don't have to modify the widget to work for categories, I'm comfortable reducing this estimate by 2h.

**Comment by benlk** [comment]
:deployed: and configured, visible at http://staging.calhealthreport.flywheelsites.com/2020/05/19/this-crisis-has-led-to-more-equitable-policies-that-california-should-keep/

**Comment by MirandaEcho** [comment]
Are we able to switch this to the more standard Largo related stories styling (side by side, with photo, headline, etc?)

**Comment by benlk** [comment]
![Screen Shot 2020-07-30 at 17 14 27](https://user-images.githubusercontent.com/1754187/88975426-27014000-d288-11ea-9269-17575c9369ef.png)

There is the option for photo and headline, but that will require additional styling. Shall I go ahead and do that in the remaining hour of this ticket?

**Comment by MirandaEcho** [comment]
Yes, please. Let's try to match the styling for the one that used to be at the bottom (see above) as much as possible, with the addition of a photo thumbnail.

**Comment by benlk** [comment]
This work is tacked onto https://github.com/INN/theme-calhealthreport/pull/27 because of the fix it contains for https://github.com/INN/theme-calhealthreport/pull/24#discussion_r463130345
